### PR TITLE
Fix navigation bug on iOS

### PR DESCRIPTION
**Pull request by jumaallan**
Story: N/A

## Summary

Fixes navigation issues on Flutter iOS side

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: ios/Classes/SmileIDBiometricKYC.swift [hunk 1]
```diff
@@ -50,9 +50,19 @@ class SmileIDBiometricKYC : NSObject, FlutterPlatformView, BiometricKycResultDel
         childViewController.view.frame = frame
         childViewController.view.autoresizingMask = [.flexibleWidth, .flexibleHeight]
         _view.addSubview(childViewController.view)
-        let rootViewController = UIApplication.shared.windows.first?.rootViewController
-        rootViewController?.addChild(childViewController)
-        _childViewController = childViewController
+
+        if let navigationController = UIApplication.shared.windows.first?.rootViewController as? UINavigationController,
```
Comment (tobitech) [inline ios/Classes/SmileIDBiometricKYC.swift:54]: I think we should be able to refactor the functionality into a function that we can then reuse for the different Job types views. That way if we need to modify the functionality we will only have to do it in one place.

Comment (vanshg) [reply ios/Classes/SmileIDBiometricKYC.swift:54]: Agreed

**Review by tobitech — APPROVED** [review]
Tested this and worked fine!
